packages/client-sdk-nodejs/src/internal/control-client.ts

```diff
@@ -182,6 +195,36 @@ export class ControlClient {
     });
   }
 
+  public async generateApiToken(
+    sessionToken: string
```

Comment:
when we start allowing generation of tokens to expire, what will the method signatures look like? Want to make sure we don't have any breaking changes.

Reply:
That's a good call. I can add an optional `validUntilSeconds` as a parameter.